.github/workflows/ci.yml

```
@@ -89,6 +89,39 @@ jobs:
           git diff
           [[ -z "$(git status --porcelain)" ]]
 
+  build-arm64:
+    strategy:
+      fail-fast: false
+    runs-on: ubuntu-20.04
+    timeout-minutes: 60
+    steps:
+    - uses: actions/checkout@v3.2.0
+    - name: Build ARM64
+      uses: uraimo/run-on-arch-action@v2.5.0
+      with:
+        arch: aarch64
+        distro: bullseye
+        run: |
+          touch /etc/apt/sources.list.d/backports.list
+          echo "deb http://deb.debian.org/debian stretch-backports main" >> /etc/apt/sources.list.d/backports.list
+          apt-get update
+          apt-get upgrade
+          apt-get install -y curl clang cmake make protobuf-compiler
+          apt-get install -y -t stretch-backports libgrpc++-dev libgrpc++1 libgrpc6 libgrpc-dev protobuf-compiler-grpc 
+          curl -sSL https://dot.net/v1/dotnet-install.sh --output dotnet-install.sh
+          chmod +x ./dotnet-install.sh
+          ./dotnet-install.sh -c 6.0 --install-dir /usr/share/dotnet --no-path
```

```suggestion
          ./dotnet-install.sh -channel 6.0.xxx --install-dir /usr/share/dotnet
```

Could try this one. Seems it doesn't know about the installed dotnet versions.

---

Will do, cheers :)